content/advanced/fine-grained-reactivity.mdx

```diff
@@ -45,7 +48,7 @@ Here, `count` serves as the getter, and `setCount` functions as the setter.
 
 ### Effects
 
-Effects are operations triggered by changes in one or more signals they depend on.
+Effects are operations triggered by changes in one or more of the signals they depend on.
```

Comment:
"+" makes me think of being an "operation", calling effects "operations" sounds weird to me. Like just saying "function", or "computation" maybe, seems better.

Also this is worded a bit strangely, like it's not the change in the signal that triggers the effect per se, in some sense it is I guess, but it seems better to say that the signal triggers the effect when the signal points to a different value